.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,9 @@
-### Description
+<!-- Read this before open de PR: https://github.com/FlamingoFiestaStudio/OlympusGoneWild/wiki/ProjectBestPractices#pull-requests -->
```

Comment:
you should add that in case of not having image or etc the person should type **N/A** in this PR template

Reply:
makes sense